### ISSUE TITLE
Conflicts with MarkdownEditing color scheme

### ISSUE THREAD

**Issue by kylebebak** [Monnoroch/ColorHighlighter#399]
Basically, #191 is back.

MacOS, build 3143. Nothing is printed to the console.

**Comment by Monnoroch** [comment]
Yes, inline highlighting should probably be disabled https://github.com/Monnoroch/ColorHighlighter#inline-color-highlighting-works-incorrectly.

**Comment by kylebebak** [comment]
I don't want to disable inline color highlighting, and `ColorHighlighter` is playing just fine with SublimeLinter, so instead I just enabled ColorHighlighter only for the extensions I code in, and not in my `.txt` and `.md` files.

This isn't a perfect solution but it will do. Definitely close this if you want.
